stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_threads_footnote.xml

```diff
@@ -1,39 +1,108 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/threadsFootnoteContainer"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:gravity="bottom"
     android:orientation="horizontal"
-    tools:showIn="@layout/stream_ui_item_message_plain_text"
     >
 
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/bottomLine"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_end="4dp"
+        />
+
+
     <androidx.appcompat.widget.AppCompatImageView
         android:id="@+id/threadsOrnamentLeft"
         android:layout_width="17dp"
         android:layout_height="40dp"
         android:src="@drawable/stream_ui_thread_replies_ornament_left"
         app:tint="@color/stream_ui_grey_whisper"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintBottom_toBottomOf="@id/bottomLine"
+        tools:visibility="gone"
+        />
+    
+    <io.getstream.chat.android.ui.avatar.AvatarView
```

Comment:
In the design, the user avatars are stacked right-to-left here (the left one is on top), and the top one has a white border

![image](https://user-images.githubusercontent.com/12054216/104822174-f9618080-5840-11eb-8fe5-3635ca116a79.png)

Are planning to make those adjustments, or is our implementation different on purpose?

Reply:
Thanks for such nice catch! I didn't notice it. Fixed

Reply:
The last visual glitch here is that with the border added, the avatar with the border is smaller than the other one, as the border is drawn within the original size of the `AvatarView`:

![image](https://user-images.githubusercontent.com/12054216/104928518-0a2a0780-59a3-11eb-8d77-ec159cfde5b8.png)

We could try to increase its size here to match what the border clips off, but we should probably revisit the border stroke implementation inside `AvatarView` itself instead. Could go into another PR / Jira task though.

Reply:
Made 18 dp size and 1 dp stroke by default for all avatars